src/Features/Blockcore.Features.Miner/MiningFeature.cs

```diff
@@ -265,6 +265,7 @@ public static IFullNodeBuilder AddPowPosMining(this IFullNodeBuilder fullNodeBui
                         services.AddSingleton<BlockDefinition, PosPowBlockDefinition>();
                         services.AddSingleton<MinerSettings>();
                         services.AddSingleton<INavigationItem, StakeNavigationItem>();
+                        services.AddSingleton<INavigationItem, MineNavigationItem>();
```

Comment:
build is broken I think you forgot some files

Reply:
[Update NavItem.cs](https://github.com/block-core/blockcore/pull/381/commits/498b62768faf3e5571823871609dd37964ab776c)